native/sapp-wasm/js/gl.js

```diff
@@ -1170,6 +1170,18 @@ var importObject = {
             } else {
                 document.exitPointerLock();
             }
+        },
+        sapp_set_fullscreen: function(fullscreen) {
+            if (!fullscreen) {
+                document.exitFullscreen();
+            } else {
+                canvas.requestFullscreen();
```

Comment:
I did not really tried it myself so it is just a question - does the browser send resize event after going fullscreen? From quick googling everyone do some things with canvas size after requestFullscreen, but if it works without - good for us!

Reply:
I didn't notice any weird behavior, it seems like it just works. One thing to note though, is that if you go fullscreen in windows, and then resize the window, you get the classic `borderless` affect that is in a lot of games.